client/src/App.js

```diff
@@ -14,6 +15,8 @@ function App() {
      update the value of this state.  Null is an initial state argument, and this indicates we  don't have the value needed 
      during the first render. */
   const [data, setData] = React.useState(null);
+  const [color, setColor] = React.useState("#47C4AF");
+  const [hidden, setHidden] = React.useState("false");
```

Comment:
Do you mind adding a couple comments on these two lines explaining what they do?